.decent_ci-Windows.yaml

```diff
@@ -1,22 +1,6 @@
 compilers:
-  - name: Visual Studio
```

Comment:
This 32-bit build was only done in packaging, and we aren't using Decent for packaging anymore, so this one goes away.